app/src/main/res/values/strings.xml

```diff
@@ -106,4 +106,6 @@
     <string name="open_source_licenses">Open source licenses</string>
     <string name="view_on_github">View project on GitHub</string>
     <string name="legal">Imprint and privacy</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
I guess, you forgot to remove this one :)

Reply:
Looks like one of the templates added it and I later thought it wasn't my addition